project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "3.0.0")
 addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.5.3")
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.7.0")
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.0.1")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.1.0")
```

Comment:
Updated there were errors with the https://github.com/alexarchambault/case-app/pull/202/files#diff-2509e699b354635ff26b61546c07e5d5R10 when running js tests.

Can either revert this version bump and make the tests JVM specific, or keep as is.